apps/OpenSpace/ext/launcher/src/sgctedit/windowcontrol.cpp

```diff
@@ -720,17 +749,55 @@ sgct::config::Window WindowControl::generateWindowInformation() const {
     vp.position = sgct::vec2(0.f, 0.f);
     vp.size = sgct::vec2(1.f, 1.f);
     vp.projection = generateProjectionInformation();
+    window.viewports.clear();
     window.viewports.push_back(vp);
     
     window.isDecorated = _windowDecoration->isChecked();
-    if (window.isFullScreen) {
-        window.monitor = _monitor->currentIndex();
-    }
-
     if (!_windowName->text().isEmpty()) {
         window.name = _windowName->text().toStdString();
     }
-    return window;
+}
+
+void WindowControl::setProjectionPlanar(float hfov, float vfov) {
+    _planar.fovH->setValue(hfov);
+    _planar.fovV->setValue(vfov);
+    _projectionType->setCurrentIndex(static_cast<int>(ProjectionIndices::Planar));
```

Comment:
If this use is the only for the `ProjectIndices`, they shouldn't be an `enum class`, but defined with separate `constexpr int` instead, in my view

Reply:
The `ProjectionIndices` struct is used in other places as well. I agree that the casts are a bit verbose, but having these defined in an `enum` feels more readable and expected that individually assigning int values.